mission-control-client/src/components/auth/Login.js

```diff
@@ -1,100 +1,70 @@
-import React, { useState } from "react";
-import { Button, Form } from "semantic-ui-react";
-import { useHistory } from "react-router";
+import React from "react";
 import axios from "axios";
-import validator from "validator";
-import { makeInputs } from "./utils";
+import {Form, Field, withFormik} from "formik";
+import {useHistory} from "react-router-dom";
+import * as Yup from "yup";
+const URL = "http://mission-control-be-dev.us-east-1.elasticbeanstalk.com/api/auth/admin/login";
 
-export default function() {
-  let history = useHistory();
-  let error = false;
-  const defaultState = {
-    email: {
-      type: "email",
-      value: "",
-      error,
-      message: "Please, enter valid email"
-    },
-    password: {
-      type: "password",
-      value: "",
-      error,
-      message: "Please, enter password between 8-16 characters"
-    }
-  };
-
-  const [state, setState] = useState(defaultState);
-
-  const Inputs = () =>
-    Object.keys(state).map(property => makeInputs(property, state, setState));
-
-  function handleSubmit(e) {
-    e.preventDefault();
-    const email = state.email.value;
-    const password = state.password.value;
+function LoginForm({
+    errors,
+    touched,
+    values,
+    handleSubmit,
+    status
+}) {
+    const history = useHistory();
+    return (
+        <Form
+            history={history}
```

Comment:
@zoverlvx what's the purpose of this prop?

Reply:
I'm taking the history hook from react-router and passing it into the form so I can then use it in the handleSubmit. At least this is how it should be working in theory.